src/sentry/grouping/ingest/config.py

```diff
@@ -17,6 +17,17 @@
 
 Job = MutableMapping[str, Any]
 
+# We are moving all projects off these configuration without waiting for events
+CONFIGS_TO_DEPRECATE = []
+
+
+def upgrade_deprecated_configs():
```

Comment:
Later I will be adding a script that calls this code.

Reply:
So will this only be called from a script? If so, can we please put it into `grouping/utils.py`, since it's not called during ingest?

Reply:
`_auto_update_grouping` is called during ingestion via `update_grouping_config_if_needed`

Reply:
I'm going to push back. All the code that is needed is here.

I can remove this function and put it directly on the script (which would make more sense).

Reply:
> `_auto_update_grouping` is called during ingestion via `update_grouping_config_if_needed`

Yup. My thought was more, "it's going to be used by both ingest and this script-only function, so it should be in a neutral location."

But anyway, if the other function is only going to be used in the script, then yeah, I think moving it there is a better idea than putting it in `utils`.